examples/pg_devsetup/docker-compose.yml

```diff
@@ -6,8 +6,20 @@ secrets:
 
 services:
 
+  postgres-init:
+    image: postgres:17
+    environment:
+      PGDATA: /var/lib/postgresql/data
+    volumes:
+      - "./volumes/postgres/data:/var/lib/postgresql/data"
+      - "./pg-init.sh:/docker-entrypoint-initdb.d/pg-init.sh:ro"
+    entrypoint: [ "/bin/bash", "./docker-entrypoint-initdb.d/pg-init.sh" ]
+
```

Comment:
why is the bootstrap not there anymore?
we need this and should demonstrate how it works, it should not require 2 services to do this, this breaks all localdevs and testing installations using docker-postgres